src/renderer/modules/components/TextArea.tsx

```diff
@@ -0,0 +1,33 @@
+import { filters, waitForModule } from "../webpack";
```

Comment:
Isn't this the same as TextInput?

Reply:
Not really. TextInput should be made to accept small values, TextArea can be longer (like descriptions) and of course a different customization props than TextInput. I don't know if it would be useful to keep this component built-in.

Reply:
I mean that I think it's the same component as input (same selector and everything). Is there just an option to make it a textarea vs input? Or am I missing something.

Reply:
Nope, they are two different components, nothing in common between them.